source/devices/AM62X/android/Release_Specific_Release_Notes.rst

```diff
@@ -57,17 +57,17 @@ SDK Components and Versions
 +------------------------------------+-------------------------------------------------------------------------------+
 | **Component**                      |  **Version**                                                                  |
 +====================================+===============================================================================+
-| **Android**                        | Android 14 / Android 14 Car                                                   |
+| **Android**                        | Android 15 / Android 15 Car                                                   |
 +------------------------------------+-------------------------------------------------------------------------------+
-| **Linux Kernel**                   | Linux 6.1.78 (6.6.30 preview available)                                       |
+| **Linux Kernel**                   | Linux 6.6.56                                                                  |
 +------------------------------------+-------------------------------------------------------------------------------+
 | **Linux Kernel Toolchain**         | clang-r487747c                                                                |
 +------------------------------------+-------------------------------------------------------------------------------+
-| **U-boot**                         | 2023.04 (2024.04 preview available)                                           |
+| **U-boot**                         | 2024.04                                                                       |
 +------------------------------------+-------------------------------------------------------------------------------+
 | **GCC Toolchain**                  | GNU Toolchain for the A-profile Architecture 13.2 (rel1)                      |
```

Comment:
@makohoek 
Just curious, 

Is there a particular for this to be 13.2 & not 13.3 similar to Yocto - https://software-dl.ti.com/processor-sdk-linux/esd/AM62X/10_00_07_04/exports/docs/devices/AM62X/linux/Release_Specific_Release_Notes.html#what-s-new

Reply:
Fixed in v2.

(In parallel, i'm working on patching https://git.ti.com/cgit/android/build-bootloaders/tree/utils.sh?h=d-android15-release#n51 but I've confirmed that it builds fine already)